conda-recipes/aif360-feedstock/.ci_support/linux_noarch.yaml

```diff
@@ -0,0 +1,10 @@
+compiler:
+- gcc
+c_compiler_version:
+- '8'
+channel_sources:
+- conda-forge, defaults
```

Comment:
Let's put defaults ahead of conda-forge here. Or drop conda-forge altogether if its not needed.